docs/ctapipe_api/reco/ImPACT.rst

```diff
@@ -98,7 +98,7 @@ examples directory soon.
 
    def reconstruct_ImPACT():
 
-        source = hessio_event_source(filename)
+        source = hessio_EventSource(filename)
```

Comment:
Probably needs correcting

Reply:
That needs removal, it is outdated docs.

Reply:
Wait. This looks like an accidentally commited file. Shouldn't the API be autogenerated and not be included in git?

Reply:
Actually it does not really matter. Event If I replaced `hessio_event_source` with `EventSource`, it wouldn't fix the docs example. And I don't see myself in the position to fix the impact docs in this PR.